src/coreclr/jit/gentree.cpp

```diff
@@ -28145,6 +28145,14 @@ void GenTreeHWIntrinsic::Initialize(NamedIntrinsic intrinsicId)
 
 #if defined(TARGET_ARM64)
             case NI_ArmBase_Yield:
+            case NI_Sve_PrefetchBytes:
```

Comment:
I believe you're missing these entries in `fgdiagnostic.cpp` as well, you'll note that it's one of the other places that handles `NI_ArmBase_Yield` if you grep the repo for it.

Reply:
ah, indeed.